js/src/ui/MethodDecoding/methodDecoding.js

```diff
@@ -120,6 +120,18 @@ class MethodDecoding extends Component {
         <span className={ styles.highlight }>
           { gas.toFormat(0) } gas ({ gasPrice.div(1000000).toFormat(0) }M/<small>ETH</small>)
         </span>
+        {
+          transaction.gasUsed
+            ? (
+              <span>
+                <span>used</span>
```

Comment:
Would have made sense to i18n these additions, but no matter, busy with this actual file in my conversion branch, so will be ok.